tests/MongoDB.Driver.Core.Tests/Core/Operations/ChangeStreamCursorTests.cs

```diff
@@ -392,7 +392,10 @@ public void Dispose_can_be_called_more_than_once()
         [Theory]
         [ParameterAttributeData]
         public void MoveNext_should_call_Resume_after_resumable_exception(
-            [Values(typeof(MongoConnectionException), typeof(MongoNotPrimaryException), typeof(MongoCursorNotFoundException))] Type resumableExceptionType,
+            [Values(
+                typeof(MongoConnectionException), // network error
+                typeof(MongoNotPrimaryException),
+                typeof(MongoNodeIsRecoveringException))] Type resumableExceptionType,
```

Comment:
The codes for this error MongoNodeIsRecoveringException correspond to the provided in the spec. But this error can also be generated for these error messages:
https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/Core/Misc/ExceptionMapper.cs#L145-L146 This is not described in the spec. But I guess it's just a different way to specify codes from here: https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Driver.Core/Core/Misc/ExceptionMapper.cs#L132-L136

Reply:
cc: @rstam

Reply:
OK I guess?

Reply:
I think so, I just wanted to have another opinion about it

Reply:
I think so, I just wanted to have another opinion about it